PortalForecasts.R

```diff
@@ -127,15 +127,17 @@ forecastall <- function(abundances,level,weather_data,weatherforecast) {
                          currency="abundance",model="Forecast", level=level, species="total", estimate=model01$mean, 
                          LowerPI=model01$lower[,which(model01$level==90)], UpperPI=model01$upper[,which(model01$level==90)])
   forecasts01[sapply(forecasts01, is.ts)] <- lapply(forecasts01[sapply(forecasts01, is.ts)],unclass)
-  
+  aic_weights = aic_weights %>%
```

Comment:
This object doesn't actually contain any AIC weights yet, does it? Maybe the data frame could be renamed to something like "model_summaries" that's more accurate throughout the lifetime of the object?